src/op_mode/clear_dhcp_lease.py

```diff
@@ -69,5 +70,8 @@ def is_ip_in_leases(address):
     if not ask_yes_no(f'This will restart DHCP server.\nContinue?'):
         exit(1)
     else:
+        if commit_in_progress():
+            print('Can not restart until commit in progress')
```

Comment:
I would do this check way earlier upon initial invokation of the script and would use a more verbose error message like: `Can not clear DHCP lease(s) as a commit is still in progress`. This comment is applicable for all further changes.

Reply:
Agree with the message.
But If we use `ask_yes_no` there is a couple of seconds to read `This will restart DHCP server.\nContinue?` and make a decision. So I use `commit_in_progress` just before `restart` 
Should I set this check earlier?

Reply:
I'd add it here: https://github.com/vyos/vyos-1x/blob/890b8eeb20998b625ec4babafc1e1bff906c347d/src/op_mode/clear_dhcp_lease.py#L69

Reply:
Done!